docs/docsite/rst/os_guide/windows_setup.rst

```diff
@@ -483,16 +493,16 @@ Ansible, select one of these installation options:
         package_params: /SSHServerFeature
         state: present
 
-* Use an existing Ansible Galaxy role like `jborean93.win_openssh <https://galaxy.ansible.com/jborean93/win_openssh>`_:
+* Download an Ansible Galaxy role for example `jborean93.win_openssh <https://galaxy.ansible.com/jborean93/win_openssh>`_:
```

Comment:
I'd prefer "Use" to "Download"

Reply:
Im not sure why i picked the word "download". I think "use" sounds better in this context.

Reply:
Changed to "Use an Ansible Galaxy role for example..."